.github/workflows/e2e-crc-okd-tests.yml

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: KubernetesClient E2E Tests (CRC-OKD)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - 'ide-config/**'
+      - '**.md'
+  schedule:
+    - cron: '0 4 * * *' # Every day at 4am
+
+concurrency:
+  group: single-instance-for-crc-okd-cluster
+  cancel-in-progress: true
+
+env:
+  IT_MODULE: kubernetes-itests
+  MAVEN_ARGS: -B -C -V -ntp -Dhttp.keepAlive=false -e
+  SHELL: /bin/bash
+
+jobs:
+  openshift-kubernetes-distribution:
+    name: CRC ${{ matrix.crc }} / OKD ${{ matrix.okd }}
+    runs-on: ubuntu-24.04
```

Comment:
Won't this run faster in macos?

Reply:
I checked with crc team about this. They had tried it in past but it didn't work due to limited resources available on macOS runner. I'll check it again and see if it's possible.